communications-handbook.md

```diff
@@ -0,0 +1,155 @@
+
+US-RSE members are encouraged to use their own personal accounts for new ideas/proposals or things under development.  The US-RSE GitHub organization is for activities of the US-RSE association rather than a collection of activities of members of the US-RSE association.
+
+The Steering Committee has ownership access to the GitHub organization; Steering Committee members can grant other individuals access to the organization and set appropriate permissions for the person. Access should only be given when needed and limited to an individual repository when possible.  People can collaborate on repos without being organizational members.
+
+If you believe that a person's access to a repository or the GitHub organization needs to be removed or changed, please send that issue to the Steering Committee (via email to contact@us-rse.org) with the reasoning. The Steering Committee may remove inactive individuals as appropriate.
+
+## Twitter
+
+[US-RSE Twitter account (@us_rse)](twitter.com/us_rse); #socialmedia channel on Slack
+
+The master password for the account should not be shared to individual people who may post to the account; instead, give access to individuals through Tweetdeck. Before giving an individual member access, please review their personal tweets to look out for any red flags that may need to be discussed, and ensure that they have read both [US-RSE Twitter Guidelines](https://docs.google.com/document/d/1JhQh0PhiXjrtkJKN9z6TGTXGdFI100t95oFyp-o5oOo/edit?usp=sharing) and our [Code of Conduct](code-of-conduct.md) and agree to both.
+
+General guidelines for what content should be posted:
+
+* Sharing US-RSE materials, events, posts, etc. 
+* Tweeting from events where US-RSE members are in attendance
+* Retweeting/liking appropriate tweets from members
+* Sharing information of general interest to our membership, including but not limited to job postings
+* Things promoting RSEs
+
+The account can be used for other purposes, such as to promote events or resources or engage with the community or individual members, but other activity should be discussed with and approved by the social media committee on Slack.  As with other activities, major initiatives should be brought to the attention of the Steering Committee. 
+
+Remove access to the account once individuals have either completed the task/time period for which they were granted access or are no longer part of the social media committee.
+
+[US-RSE Twitter Guidelines](https://docs.google.com/document/d/1JhQh0PhiXjrtkJKN9z6TGTXGdFI100t95oFyp-o5oOo/edit?usp=sharing) is intended for sharing with anyone who will be tweeting from the account.
```

Comment:
Should this stay in a google doc, or move here?

Reply:
That is a good question.  I landed on it staying in the google doc since it's not really official and meant to be sent to folks who will post to the account.  Probably depends on what our procedure is for changing things in the repo -- that is an example of a document that doesn't really need much approval to change.  I pulled out the more policy oriented parts of the twitter guidelines into the handbook, because those are things that may require more input if they are changed.

Reply:
I think it's good to have something separate that:
* has more detail that we'd want here
* includes operational details we might not want public
* is able to be more fluid, giving specifics that fit within the guidance of this document

Reply:
I should have been more clear, by "here" I was thinking in this repo, not this particular document.

Reply:
leaving in google docs for now.  We can move it here later if we think it's appropriate